lib/registration.pm

```diff
@@ -124,11 +124,12 @@ sub scc_version {
 Wrapper for SUSEConnect -p $name.
 =cut
 sub add_suseconnect_product {
-    my ($name, $version, $arch, $params) = @_;
+    my ($name, $version, $arch, $params, $timeout) = @_;
     $version //= scc_version();
     $arch    //= get_required_var('ARCH');
     $params  //= '';
-    assert_script_run("SUSEConnect -p $name/$version/$arch $params");
+    $timeout //= 90;
```

Comment:
why do you set this here? if `$timeout` is not set then `assert_script_run` should revert to default which is AFAIK also 90s

Reply:
in s390x doesn't work with 90s, if it is buggy, maybe but we don't have the test suite, this is for giving us an idea (read my last comment too)